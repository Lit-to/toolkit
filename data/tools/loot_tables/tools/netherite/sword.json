{
    "pools": [
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "weight": 1,
                    "name": "minecraft:netherite_sword",
                    "functions": [
                        {
                            "function": "set_count",
                            "count": 1
                        },
                        {
                            "function": "set_name",
                            "name": [
                                {
                                    "text": "ネザライトの剣"
                                }
                            ]
                        }
                    ]
                }
            ]
        }
    ]
}