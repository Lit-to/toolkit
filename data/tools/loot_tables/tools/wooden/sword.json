{
    "pools": [
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "weight": 1,
                    "name": "minecraft:wooden_sword",
                    "functions": [
                        {
                            "function": "set_count",
                            "count": 1
                        },
                        {
                            "function": "set_name",
                            "name": [
                                {
                                    "text": "木の剣"
                                }
                            ]
                        }
                    ]
                }
            ]
        }
    ]
}