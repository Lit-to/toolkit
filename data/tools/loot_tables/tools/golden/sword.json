{
    "pools": [
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "weight": 1,
                    "name": "minecraft:golden_sword",
                    "functions": [
                        {
                            "function": "set_count",
                            "count": 1
                        },
                        {
                            "function": "set_name",
                            "name": [
                                {
                                    "text": "金の剣"
                                }
                            ]
                        }
                    ]
                }
            ]
        }
    ]
}